docs/integrations/microsoft-azure/azure-storage.md

```diff
@@ -52,13 +56,99 @@ Capacity metrics are currently not supported via Diagnostic Settings.
+   - `tenant_name` - This field is tagged at the collector level and users can get the tenant name using the instructions here https://learn.microsoft.com/en-us/azure/active-directory-b2c/tenant-management-read-tenant-name#get-your-tenant-name
+   - `location` - The region which the resource name belongs to
+   - `subscription_id` - Id associated with a subscription where resource is present
+   - `resource_group` - The resource group name where the Azure resource is present.
+   - `provider_name` - Azure resource provider name (for  ex Microsoft.Storage)
+   - `resource_type` - Azure resource type (for ex storageaccounts)
+   - `resource_name` - The name of the resource (for ex storage account name)
+   - `service_type` - Services that can be accessed from within a azure resource (Ex table service in storage account) 
+
+3. If not present, create it. Learn how to create and manage fields [here](/docs/manage/fields/#manage-fields).
+
+### Configure Field Extraction Rules
+Create a Field Extraction Rule (FER) for Azure Storage by following the instructions [here](/docs/manage/field-extractions/create-field-extraction-rule/).
+
+* **Activity Logs Location Extraction FER**
+
+   ```sql
+   Rule Name: AzureLocationExtractionFER
+   Applied at: Ingest Time
+   Scope (Specific Data): tenant_name=*
+   ```
+
+   ```sql title="Parse Expression"
+   json "location", "properties.resourceLocation", "properties.region" as location, resourceLocation, service_region nodrop
+   | replace(toLowerCase(resourceLocation), " ", "") as resourceLocation
+   | if (!isBlank(resourceLocation), resourceLocation, location) as location
+   | if (!isBlank(service_region), service_region, location) as location 
+   | if (isBlank(location), "global", location) as location
+   | fields location
+   ```
+
+* **Resource ID Extraction FER**
+
+   ```sql
+   Rule Name: AzureResourceIdExtractionFER
+   Applied at: Ingest Time
+   Scope (Specific Data): tenant_name=*
+   ```
+
+   ```sql title="Parse Expression"
+   json "resourceId"
+   | toUpperCase(resourceId) as resourceId
+   | parse regex field=resourceId "/SUBSCRIPTIONS/(?<subscription_id>[^/]+)" nodrop
+   | parse field=resourceId "/RESOURCEGROUPS/*/" as resource_group nodrop
+   | parse regex field=resourceId "/PROVIDERS/(?<provider_name>[^/]+)" nodrop
+   | parse regex field=resourceId "/PROVIDERS/[^/]+(?:/LOCATIONS/[^/]+)?/(?<resource_type>[^/]+)/(?<resource_name>.+)" nodrop
+   | parse regex field=resource_name "(?<parent_resource_name>[^/]+)(?:/PROVIDERS/[^/]+)?/(?<service_type>[^/]+)/?(?<child_service_name>.+)" nodrop
+   | if (isBlank(parent_resource_name), resource_name, parent_resource_name) as resource_name
+   | fields subscription_id, location, provider_name, resource_group, resource_type, resource_name, service_type
+   ```
+### Configure metric rules
+* **Azure Observability Metadata Extraction Metric Rule Service Level**
+```sql
+Rule Name: AzureObservabilityMetadataExtractionMetricRuleServiceLevel
```

Comment:
AzureObservabilityMetadataExtractionMetricRuleServiceLevel => AzureObservabilityMetadataExtractionServiceLevel

Reply:
We should also mention if this rule is already present then no need to recreate it.